fetch.bs

```diff
@@ -1417,42 +1417,45 @@ Unless stated otherwise, it is unset.
 "<code>no-cache</code>", "<code>force-cache</code>", or
 "<code>only-if-cached</code>". Unless stated otherwise, it is "<code>default</code>".
 
+<p>Let <var>httpCache</var> be the result of <a>determining the HTTP cache partition</a> for
```

Comment:
This step doesn't seem to be located in the correct place. What uses this httpCache variable? It needs to be inside the fetch algorithm, not standing alone at top level. Then you can pass it another variable representing the current request as an argument, instead of passing it the general concept of a request.

Reply:
Ah, I see, you do include it below. Yeah, then just remove all changes to this section. This section is defining concepts and doesn't need to get into these details.

Reply:
Removed